packages/@okta/i18n/src/properties/login.properties

```diff
@@ -1012,17 +1012,28 @@ oie.email.return.link.expired.title = Verify with your email
 oie.profile.additional.title = Additional Profile information
 oie.profile.additional.secondemail.subtitle = Use a second email to <$1> recover your account </$1> in case you become locked out. This email must be verified after setup.
 ## Webauthn
+signinWithPasskeys.button = Sign in with a passkey
+signin.passkeys.error = Could not sign in with a passkey. Try again, or sign in another way to add a passkey to your account.
 oie.webauthn.label = Security Key or Biometric Authenticator
+oie.webauthn.passkeysRebrand.passkeys.label = Passkey
 oie.webauthn.description = Use a security key or a biometric authenticator to sign in
+oie.webauthn.passkeysRebrand.passkeys.description = Use a passkey to sign in with biometrics or a security key.
 oie.enroll.webauthn.title = Set up security key or biometric authenticator
+oie.enroll.webauthn.passkeysRebrand.passkeys.title = Set up a passkey
+# {0} is the authenticator name
+oie.enroll.webauthn.passkeysRebrand.custom.title = Set up {0}
 oie.enroll.webauthn.instructions = You will be prompted to register a security key or biometric authenticator (Windows Hello, Touch ID, Face ID, etc.). Follow the instructions to complete set up.
 oie.enroll.webauthn.uv.required.instructions = Biometric verification or a PIN is required to setup this authenticator.
 oie.enroll.webauthn.instructions.edge = Note: If you are enrolling a security key and Windows Hello or PIN is enabled, you will need to select 'Cancel' in the prompt before continuing.
 oie.enroll.webauthn.save = Set up
 oie.webauthn.error.not.supported = Security key or biometric authenticator is not supported on this browser. Contact your admin for assistance.
 oie.webauthn.error.invalidPasskey = Invalid passkey.
 oie.verify.webauth.title = Verify with Security Key or Biometric Authenticator
+oie.verify.webauth.passkeysRebrand.passkeys.title = Verify with a passkey
+# {0} is the authenticator name
+oie.verify.webauth.passkeysRebrand.custom.title = Verify with {0}
```

Comment:
NOTE: @LazarVuckovic-okta watch out for the case/condition where the number params mismatch which will throw a l10n error. The translation itself is fine.